thirdparty/CMakeLists.txt

```diff
@@ -188,6 +188,14 @@ if(ENABLE_ONEDNN_FOR_GPU)
     build_onednn_gpu()
 endif()
 
+#
+# nlohmann json
+#
+if(ENABLE_SAMPLES OR NGRAPH_PDPD_FRONTEND_ENABLE OR NGRAPH_ONNX_FRONTEND_ENABLE OR NGRAPH_TF_FRONTEND_ENABLE)
```

Comment:
Do we need this json library in samples?

Reply:
yes, we are planning to use it in benchmark_app
previously we parsed json via OpenCV, but as far as I know, we abandon the dependency on this library.